build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.2"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.gms:google-services:4.3.5'
```

Comment:
There's a comment stating  "NOTE: Do not place your application dependencies here; they belong in the individual module build.gradle files" . Is this as issue for line 11 as in do we need another build.gradle file for this module?